src/wasm/kotlin/java/arcs/entity-class.kt

```diff
@@ -0,0 +1,117 @@
+// This is a reference for the entity classes generated by schema2wasm, based on the schema:
+//
+// particle Test
+//   data: reads {num: Number, txt: Text, lnk: URL, flg: Boolean, ref: &{val: Text}}
```

Comment:
It would be best to make this a file-level KDoc block, and place it between the `package` line and the start of the class definitions.

Reply:
Also, the file name: `entity-class.kt` should probably be `EntityClass.kt`

Reply:
Bear in mind this is a reference file, not intended to be compiled and used. Its purpose is to show the output of the schema2wasm tool.